uui/components/overlays/Alert.tsx

```diff
@@ -27,10 +30,12 @@ export interface AlertProps extends IHasChildren, IHasCX, IHasRawProps<React.HTM
     onClose?(): void;
     /** An optional icon to show on the left of the alert */
     icon?: Icon;
+    /** List of size, you can use {name: 'compact'} and by default its {name: 'default'} */
+    size?: alertSize[];
```

Comment:
Let's use here height number values for size prop, as we have in our other component: `size: '36' | '48'`

Reply:
ok

Reply:
Done, 
any changes?